shared/chat/conversation/messages/wrapper/index.tsx

```diff
@@ -478,7 +478,16 @@ class _WrapperMessage extends React.Component<Props & Kb.OverlayParentProps, Sta
         child = <SystemInviteAccepted key="systemInviteAccepted" message={message} />
         break
       case 'systemSBSResolved':
-        child = <SystemSBSResolved key="systemSbsResolved" message={message} />
+        if (this.props.youAreAuthor) {
```

Comment:
This seems like a mildly odd place to have this logic, but I don't have any better suggestions.. 🤷

Reply:
Yeah, I thought the same thing but I couldn't come up with a better place for it :/